editor/src/components/canvas/canvas-strategies/strategies/grid-draw-to-insert-strategy.tsx

```diff
@@ -171,7 +171,7 @@ const gridDrawToInsertStrategyInner =
               ),
               updateHighlightedViews('mid-interaction', [targetParent]),
             ],
-            [targetParent],
+            [],
```

Comment:
Are you sure we no longer need to pass in the `targetParent` here? Or was this maybe an old investigation that was accidentally included?

Reply:
This is for the stage where insertion has been activated but not started, so there's no need to re-render the parent.